Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:16
+
+LABEL maintainer="me@jqshuv.xyz"
+LABEL org.opencontainers.image.source https://github.com/ZynerOrg/xyter
+LABEL org.opencontainers.image.description "An multi-purpose discord.js bot."
+LABEL org.opencontainers.image.licenses=GPL-3.0-only
+
+
+WORKDIR /app
+
+LABEL maintainer="me@jqshuv.xyz"
+LABEL org.opencontainers.image.source https://github.com/ZynerOrg/xyter
+LABEL org.opencontainers.image.description "An multi-purpose discord.js bot."
+LABEL org.opencontainers.image.licenses=GPL-3.0-only
+
+COPY package* .
+RUN npm install
+
+COPY . .
+RUN mv src/config_docker src/config
+
+RUN npx tsc
+
+RUN npx tsc -v > /app/tsc.log
+RUN npm -v > /app/npm.log
+RUN node -v > /app/node.log
```

Comment:
This should be inside of ``logs/``

Reply:
Yes, first it was only to send you the versions, due to the error. But I can do this of course.

Reply:
Maybe even better to post it into ``logs/package.versions`` or something. This way it's easy to help users that has problems. But remove it if you want to, since I wont publish a non working image 😊